microsoft-edge/devtools-guide-chromium/customize/localization.md

```diff
@@ -22,16 +22,53 @@ Your Developer Tools matches your browser language by default, if it is one of t
 * Russian – &#1088;&#1091;&#1089;&#1089;&#1082;&#1080;&#1081;
 * Spanish - espa&#241;ol
 
-DevTools in Chinese:
 
-:::image type="content" source="../media/localization-zh.msft.png" alt-text="DevTools in Chinese." lightbox="../media/localization-zh.msft.png":::
+The DevTools UI in Chinese:
+
+![The DevTools UI in Chinese.](../media/localization-zh.msft.png)
+
+_To zoom: right-click > **Open image in new tab**._
```

Comment:
Is this italicized because it is optional? Just looks odd to me.

Reply:
@vchapel - Removed these tips per captain b review thread in PR 1665.